student.rb

```diff
@@ -1,8 +1,6 @@
-require_relative 'person'
+require './person'
 
 class Student < Person
-  attr_accessor :classroom
```

Comment:
- [ ] Hello! 😊 It's super important to have getters for the classroom 🏫 because they give you a way to easily check and access the current status of the classroom. 📊 This is crucial because having this access helps you smoothly assign a specific student to the classroom without any hiccups or mistakes. 🎯 Getters act like your trusty information detectives, making sure you have the right details about the classroom's state, which is 🧐 pretty crucial for maintaining data integrity and making well-informed decisions when you're juggling classroom-related tasks. 📚💡